coordinator/src/test/java/org/apache/uniffle/coordinator/QuotaManagerTest.java

```diff
@@ -116,13 +116,15 @@ public void testCheckQuota() throws Exception {
     final boolean icCheck =
         applicationManager.getQuotaManager().checkQuota("user4", String.valueOf(i1));
     registerThread.join();
-    assertTrue(icCheck);
+    assertFalse(icCheck);
```

Comment:
Could you add a case for `rss.coordinator.quota.default.app.num = 5`?

Reply:
sure

Reply:
I'll keep current testCheckQuota logic for test `rss.coordinator.quota.default.app.num = 5`, add a new `testCheckQuotaWithDefault` function to test  default value -1.